suite-native/state/src/store.ts

```diff
@@ -24,30 +27,27 @@ if (__DEV__) {
     middlewares.push(reduxFlipperDebugger);
 }
 
-export const transactionsReducer = prepareTransactionsReducer(extraDependencies);
-export const accountsReducer = prepareAccountsReducer(extraDependencies);
-export const fiatRatesReducer = prepareFiatRatesReducer(extraDependencies);
-export const blockchainReducer = prepareBlockchainReducer(extraDependencies);
+const rootPersistConfig = {
+    key: 'root',
+    storage: reduxStorage,
+    whitelist: ['appSettings', 'wallet'],
+};
 
-const walletReducers = combineReducers({
-    accounts: accountsReducer,
-    blockchain: blockchainReducer,
-    fiat: fiatRatesReducer,
-    transactions: transactionsReducer,
-});
+const persistedReducer = persistReducer(rootPersistConfig, rootReducers);
 
 export const store: Store = configureStore({
-    reducer: {
-        appSettings: appSettingsReducer,
-        devices: devicesReducer,
-        wallet: walletReducers,
-    },
+    reducer: persistedReducer,
     middleware: getDefaultMiddleware =>
         getDefaultMiddleware({
             thunk: {
                 extraArgument: extraDependencies,
             },
+            serializableCheck: {
+                ignoredActions: [FLUSH, REHYDRATE, PAUSE, PERSIST, PURGE, REGISTER],
```

Comment:
Why do we need this? Can you please add comment to code?

Reply:
https://redux-toolkit.js.org/usage/usage-guide#use-with-redux-persist

Reply:
Thank you, Leo :). I added the comment to be more clear what this is about.